DSharpPlus.Interactivity/InteractivityConfiguration.cs

```diff
@@ -45,7 +46,22 @@ public sealed class InteractivityConfiguration
         /// <summary>
         /// Emojis to use for pagination
         /// </summary>
-        public PaginationEmojis PaginationEmojis { internal get; set; } = new PaginationEmojis();
+        public PaginationEmojis PaginationEmojis { internal get; set; } = new();
+
+        /// <summary>
+        /// Buttons to use for pagination.
+        /// </summary>
+        public PaginationButtons PaginationButtons { internal get; set; } = new();
+
+        /// <summary>
+        /// Whether interactivity should ACK buttons that are pushed. Setting this to <see langword="true"/> will also prevent subsequent event handlers from running.
+        /// </summary>
+        public bool ACKPaginationButtons { internal get; set; }
```

Comment:
AckPaginationButton, I guess?

Reply:
+1, keeping ACK in all caps is weird. Kiritsu's suggestion makes more sense.